components/org.wso2.carbon.identity.api.server.secret.management/org.wso2.carbon.identity.api.server.secret.management.common/src/main/java/org/wso2/carbon/identity/api/server/secret/management/common/SecretManagementConstants.java

```diff
@@ -72,7 +77,7 @@
 
         public String getCode() {
 
-            return ERROR_PREFIX + code;
```

Comment:
As we change this, don't we need to add prefix to the existing codes

Reply:
It already has a prefix as SECRETM. mistakenly added twice before

Reply:
resolved